src/index.js

```diff
@@ -12,6 +12,12 @@ const shell = require('shelljs')
 const pluralize = require('pluralize')
 const requireEveryTime = require('require-and-forget')
 
+// Require CJS loader to resolve:
+// https://github.com/bahmutov/find-cypress-specs/issues/228
+// https://github.com/bahmutov/find-cypress-specs/issues/222
+// https://github.com/privatenumber/tsx
+require('tsx/cjs')
```

Comment:
@samtsai I switched to this because your module was archived, this is the new version. The old one was causing a crash: https://github.com/bahmutov/find-cypress-specs/actions/runs/7881054762/job/21504058322

Reply:
oh cool, even better!